src/components/Messages/index.tsx

```diff
@@ -12,7 +12,7 @@ import PreviewList from './PreviewList';
 const Messages: NextPage = () => {
   const currentProfile = useAppStore((state) => state.currentProfile);
 
-  if (!currentProfile || !isFeatureEnabled('messages', currentProfile.id)) {
+  if (!isFeatureEnabled('messages', currentProfile?.id)) {
```

Comment:
we need to have currentProfile check, else it will not render 404 if profile is not there 🙈